Slack pack won't install on python 3.8

The slack pack won't install on 3.5dev on focal and complains first with:

```
      1 | #include "libxml/xpath.h"
        |          ^~~~~~~~~~~~~~~~
  compilation terminated.
  *********************************************************************************
  Could not find function xmlCheckVersion in library libxml2. Is libxml2 installed?
```

Installing libxml2-dev and libxslt1-dev resolves this problem, but then the following problem is seen:
```
    x86_64-linux-gnu-gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -I/usr/include/libxml2 -Isrc/lxml/includes -I/opt/stackstorm/virtualenvs/slack/include -I/usr/include/python3.8 -c src/lxml/lxml.etree.c -o build/temp.linux-x86_64-3.8/src/lxml/lxml.etree.o -w
    src/lxml/lxml.etree.c: In function __Pyx_PyCFunction_FastCall:
    src/lxml/lxml.etree.c:233279:13: error: too many arguments to function (PyObject * (*)(PyObject *, PyObject * const*, Py_ssize_t))meth
    233279 |     return (*((__Pyx_PyCFunctionFast)meth)) (self, args, nargs, NULL);
           |            ~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    src/lxml/lxml.etree.c: In function __Pyx_PyDict_Items:
    src/lxml/lxml.etree.c:4532:83: error: too many arguments to function (PyObject * (*)(PyObject *, PyObject * const*, Py_ssize_t))__pyx_umethod_PyDict_Type_items.func

```

lxml < 4.0.0 doesn't support python 3.8, and so for focal need to use newer version of lxml.

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,5 +1,9 @@
 # Change Log
 
+# 2.1.0
+
+* Upgrade lxml dependency from 3.8.0 to 4.6.5
+
 # 2.0.5
 
 * Add `blocks` from slack message to sensor `payload`

--- a/pack.yaml
+++ b/pack.yaml
@@ -7,7 +7,7 @@ keywords:
   - chat
   - messaging
   - instant messaging
-version: 2.0.5
+version: 2.1.0
 python_versions:
   - "3"
 author : StackStorm, Inc.

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,6 +1,6 @@
 requests>=2.5.0,<3.0
 websocket-client==0.44.0
 beautifulsoup4==4.6.0
-lxml==3.8.0
+lxml==4.6.5
 jinja2>=2.10.1
 slackclient==1.3.1

